client/me/account/main.jsx

```diff
@@ -84,25 +82,58 @@ const Account = React.createClass( {
 		debug( this.constructor.displayName + ' component is unmounting.' );
 	},
 
-	updateLanguage() {
-		let valueLink = this.valueLink( 'language' );
+	getUserSetting( settingName ) {
+		return this.props.userSettings.getSetting( settingName );
```

Comment:
note that we have introduced an opening for `TypeError`s when `userSettings` isn't an object with a `getSetting` value

as a function in the module scope we could generalize this in a safe way…

```js
const getSetting = ( props, name ) => invoke( props, 'userSettings.getSetting', name );

…

getSetting( this.props, 'user_login' );
```